### DISCUSSION TITLE
Designer for Prime NG v15

### DISCUSSION THREAD

**Discussion by joshnarwold-mck** [primefaces/community#109] [PrimeNG] [answered]
Hi there - it used to be that I could download the Designer code for a specific version of Prime NG. Is there a way to do that now that Designer is a github project?

My team is upgrading to Prime NG v15, and I'd like to make sure we have the correct Designer code.

**Comment by cetincakiroglu** [comment]
Hi,

Designer is now an open-source project and you can do the exact same process by just cloning the repo from github.

**Comment by onursenture** [comment, marked as answer]
Here is the link to releases of open sourced Designer: https://github.com/primefaces/primeng-sass-theme/releases

**Comment by The-Real-1** [comment]
Hi everyone,

I think there is a lot of confusion about what is called Designer.

For me  it is that : [Link to the designer](https://designer.primeng.org/#/)


I have a question about how we are supposed to use the new versions in addition with the Designer Application (Visual and Code).
We used to have a license to download the Designer project but as mentionned , it seems that the Designer is free now.
When I look at the repository (https://github.com/primefaces/primeng-sass-theme/releases) , i do not see the Designer project strucutre. Only the scss files. 

Is there a documentation explaining how we are supposed to deal with those files inside the Designer project ?
Does that means that the Designer project will no longer be released ?



Thanks a lot.

**Comment by elmota** [comment]
thats just a bunch of scss fles, not designer UI
